src/qt/pivx/mnmodel.cpp

```diff
@@ -149,4 +149,12 @@ bool MNModel::addMn(CMasternodeConfig::CMasternodeEntry* mne){
     nodes.insert(QString::fromStdString(mne->getAlias()), std::make_pair(QString::fromStdString(mne->getIp()), pmn));
     endInsertRows();
     return true;
+}
+
+bool MNModel::isMNMissing(QString mnAlias) {
+    QMap<QString, std::pair<QString, CMasternode*>>::const_iterator it = nodes.find(mnAlias);
+    if (it != nodes.end()) {
+        return it.value().second->activeState == CMasternode::MASTERNODE_MISSING;
```

Comment:
I think that here we should consider as "missing" anything that has active state `activeState != CMasternode::MASTERNODE_ENABLED`